packages/base-sheets/src/Controller/ToggleWorksheet.controller.ts

```diff
@@ -0,0 +1,109 @@
+import {
+    IRenderManagerService,
+    ISelectionTransformerShapeManager,
+    Rect,
+    Spreadsheet,
+    SpreadsheetColumnHeader,
+    SpreadsheetRowHeader,
+} from '@univerjs/base-render';
+import { Disposable, ICurrentUniverService, LifecycleStages, OnLifecycle } from '@univerjs/core';
+import { Inject } from '@wendellhu/redi';
+
+import { CANVAS_VIEW_KEY, SHEET_VIEW_KEY } from '../Basics/Const/DEFAULT_SPREADSHEET_VIEW';
+import { columnWidthByHeader, rowHeightByHeader } from '../Basics/SheetHeader';
+import { SheetSkeletonManagerService } from '../services/sheet-skeleton-manager.service';
+
+@OnLifecycle(LifecycleStages.Rendered, ToggleWorksheetController)
+export class ToggleWorksheetController extends Disposable {
```

Comment:
用 switch 这个词，Toggle 的意思是开关

```suggestion
export class SwitchWorksheetController extends Disposable {
```

Reply:
ok